lua/neotest-haskell/hspec.lua

```diff
@@ -247,7 +247,7 @@ local function get_hspec_errors(raw_lines, test_name)
         message = error_message,
       } }
     elseif pos_found then
-      error_message = error_message and error_message .. '\n' .. trimmed or trimmed
+      error_message = error_message and error_message:gsub('%s*$', '') .. '\n' .. trimmed or trimmed
```

Comment:
I would move this fix to the line above:
```lua
local trimmed = line:match('^%s*(.*)'):gsub('%s*$', '')
```
(GitHub won't let me include that line in the review for some reason)

Reply:
Oh wait, `error_message` could be `nil`.
Oh, how I love Lua...

Reply:
Let's do
```lua
local trimmed = (line:match('^%s*(.*)') or line):gsub('%s*$', '')
```
to be safe.